src/libraries/System.ComponentModel.TypeConverter/tests/TypeDescriptorTests.cs

```diff
@@ -1140,6 +1140,12 @@ public void DerivedPropertyAttribute()
             Assert.Equal("Derived", descriptionAttribute.Description);
         }
 
+        [Theory]
+        [InlineData(typeof(Version), typeof(VersionConverter))]
+        [InlineData(typeof(IComponent), typeof(ComponentConverter))]
+        public static void GetConverter_NetCoreApp(Type targetType, Type resultConverterType) =>
+            GetConverter(targetType, resultConverterType);
+
         class FooBarBase
```

Comment:
You can also delete the `SkipOnTargetFramework` in this file while you are on it. It is no longer necessary.